vividus-plugin-visual/src/main/java/org/vividus/visual/engine/VisualTestingEngine.java

```diff
@@ -41,6 +41,7 @@
     private final ScreenshotProvider screenshotProvider;
     private final IBaselineRepository baselineRepository;
 
+    private int sensitivity;
```

Comment:
`acceptableDiffPercentage` ?
`web.visual.acceptable-diff-percentage=0` - by default
and also please docs for this property